toolkit/tools/pkg/imagecustomizerlib/customizeboot.go

```diff
@@ -31,16 +31,30 @@ const (
 	selinuxConfigModeRegexSELinuxMode = 1
 )
 
-// Looks for a command with the provided name and ensures there is only 1 such command.
-// Returns the line of the found command.
-func findSingularGrubCommand(inputGrubCfgContent string, commandName string) (grub.Line, error) {
+// Looks for all occurences of a command with the provided name.
+// Returns the lines containing the command.
+func findGrubCommandAll(inputGrubCfgContent string, commandName string) ([]grub.Line, error) {
 	grubTokens, err := grub.TokenizeConfig(inputGrubCfgContent)
 	if err != nil {
-		return grub.Line{}, err
+		return nil, err
 	}
 
 	grubLines := grub.SplitTokensIntoLines(grubTokens)
 	lines := grub.FindCommandAll(grubLines, commandName)
+	if len(lines) < 1 {
+		return nil, fmt.Errorf("failed to find the '%s' command in grub config", commandName)
+	}
+
+	return lines, nil
+}
+
+// Looks for a command with the provided name and ensures there is only 1 such command.
+// Returns the line of the found command.
+func findSingularGrubCommand(inputGrubCfgContent string, commandName string) (grub.Line, error) {
```

Comment:
You can omit the `len(lines) < 1` check from this function.

Reply:
true - ok

Reply:
pending.

Reply:
fixed.